src/components/Filters/FilterModal.tsx

```diff
@@ -50,22 +48,27 @@ const defaultFilterCompareFn: FilterFieldCompareFn = (a, b) => {
 };
 
 const FilterInput = (props: FilterInputProps) => {
+	const { schema, value, onUpdate } = props;
 	const model = getDataModel(props.schema);
-
-	if (!model) {
-		return null;
```

Comment:
Maybe `if (!model) return null;` is fine to keep?
My concern is whether the filters will be able to work if we find a type for which we don't have a DataModel for, even though the Form will be able to render it.

Reply:
if model is not defined we should try to render the form in any case no ? The model is not needed to render the Form component, right ?